RNCryptor/RNDecryptor.m

```diff
@@ -53,6 +53,9 @@ - (BOOL)rnc_isEqualInConsistentTime:(NSData *)otherData;
 @implementation NSData (RNCryptor_ConstantCompare)
 
 - (BOOL)rnc_isEqualInConsistentTime:(NSData *)otherData {
+  if (!otherData) {
+    return NO;
+  }
```

Comment:
Hm, without this, this method would return YES if self was an empty NSData and otherData was nil.